tests/integration/targets/alternatives/tasks/main.yml

```diff
@@ -49,6 +49,9 @@
   # Test that path is checked: alternatives must fail when path is nonexistent
   - import_tasks: path_is_checked.yml
 
+  # Test that subcommands commands work
+  - import_tasks: subcommands.yml
+
```

Comment:
It would be great to also have some tests for `state=absent`.

Reply:
Added tests for `state=absent` and change in `priority` in c2423825eb5a9cfb96326220c72029f2e561fc7b